db/seeds.rb

```diff
@@ -65,44 +65,98 @@ def puts_records_in_db(klass)
 
 montana = State.find_or_create_by!(name: 'Montana', abbr: 'MT')
 big_horn_cty_mt = County.find_or_create_by!(name: 'Big Horn', state: montana)
-hardin_zipcode = Zipcode.first_or_create!(city: 'Hardin', county: big_horn_cty_mt, state: big_horn_cty_mt.state, code: '12345')
+hardin_zipcode = Zipcode.find_or_create_by!(city: 'Hardin', county: big_horn_cty_mt, state: big_horn_cty_mt.state, code: '12345')
+
+illinois = State.find_or_create_by!(name: 'Illinois', abbr: 'IL')
+cook_county = County.find_or_create_by!(name: 'Cook', state: illinois)
+cook_zip_60686 = Zipcode.find_or_create_by!(state: cook_county.state, county: cook_county, code: '60686', city: 'Chicago')
+cook_zip_60688 = Zipcode.find_or_create_by!(state: cook_county.state, county: cook_county, code: '60688', city: 'Chicago')
+
+lake_county = County.find_or_create_by!(name: 'Lake', state: illinois)
 
 # ---------------------------------------------
 # Businesses
 # ---------------------------------------------
 
-@business = Business.where(name: 'Happy Seedlings Childcare', user: @user_kate).first_or_create(
+@biz_happy_seedlings = Business.where(name: 'Happy Seedlings Childcare', user: @user_kate).first_or_create(
   license_type: Licenses.types.keys.first,
   county: big_horn_cty_mt,
   zipcode: hardin_zipcode
 )
 
+@biz_goslings = Business.where(name: 'Goslings Grow', user: @user_kate).first_or_create(
+  license_type: Licenses.types.keys.first,
+  county: cook_county,
+  zipcode: cook_zip_60688
+)
+@biz_lil_ducks = Business.where(name: 'Lil Baby Ducklings', user: @user_kate).first_or_create(
+  license_type: Licenses.types.keys.first,
+  county: cook_county,
+  zipcode: cook_zip_60686
+)
+
 puts_records_in_db(Business)
 
+# ---------------------------------------------
+# Approvals
```

Comment:
This seeding shouldn't be necessary, should it?  We're not referencing these approvals again, we're just creating new ones for the kids below, right?

Reply:
They were (are) required for the `.csv` file I created.  Since I'll get rid of the csv file, I'll take them back out.